hack/upload

```diff
@@ -35,5 +35,11 @@ if [[ "${DEST:0:6}" == "oss://" ]]; then
   exit 0
 fi
 
+if [[ "${DEST:0:5}" == "do://" ]]; then
+  DO_BUCKET=`echo "${DEST}" | cut -c 6-`
+  s3cmd put "*" ${SRC}/ s3://$DO_BUCKET --recursive --acl-public
```

Comment:
Can this use `PUBLIC` like the others?

Reply:
I have set the --acl-public flag set, allowing read access for everyone.

Reply:
The point is to allow callers to omit the `--acl-public` flag.

Reply:
Ah, I got that now. I didn't realize parameter substitution was happening.. I updated it and squashed the commits as well.